upgrade.md

```diff
@@ -17,6 +17,76 @@ The `storagePath` method of the `Illuminate\Contracts\Foundation\Application` in
 
     public function storagePath($path = '');
 
+<a name="mail"></a>
+### Mail
+
+<a name="symfony-mailer"></a>
+#### Symfony Mailer
+
+**Likelihood Of Impact: High**
+
+Swift Mailer is discontinued as of December 2021 and was therefore replaced with Symfony Mailer 6.0 (See https://symfony.com/blog/the-end-of-swiftmailer)
+Because of this SwiftMailer plugins and transports will no longer work with the new implementation.
+
+##### Updated Content Methods
+`send`, `html`, `text` & `plain` don't return the number of recipients anymore but an instance of `Illuminate\Mail\SentMessage` which contains the `Symfony\Component\Mailer\SentMessage` instance
+
+##### Renamed Swift methods to Symfony
+
+All Swift methods were renamed to Symfony. For example:
+
+    // BEFORE
+    $this->withSwiftMessage(function ($message) {
+        $message->getHeaders()->addTextHeader(
+            'Custom-Header', 'Header Value'
+        );
+    });
+
+    // AFTER
+    use Symfony\Component\Mime\Email;
+
+    $this->withSymfonyMessage(function (Email $message) {
+        $message->getHeaders()->addTextHeader(
+            'Custom-Header', 'Header Value'
+        );
+    });
+
+##### `Illuminate\Mail\Message`
+The Illuminate\Mail\Message class now contains an instance of Symfony\Component\Mime\Email instead of Swift_Message so all forwarding calls in userland will need to be updated.
```

Comment:
We will perhaps need an example of this.

Reply:
There were only Methods which were not documented by Laravel because, Laravel already used `from`, `subject, `cc` like the Symfony Mailer. There should probably not be many changes needed.